### PR TITLE
media-video/vcsi: new package

### PR DESCRIPTION
**Pull request by juippis**
Vcsi is a tool to create video contact sheets and Ive been using it since version 3. I see myself using this in the future and am willing to proxy-maintain it. I also added myself as proxy-maintainer to dev-python/texttable, since it is required by vcsi. 

This is my first time doing anything with python tests, but I tested vcsi and texttable and both were fine. Cjkwrap upstream only says it supports python up to 3.5, but I installed python3.6 and emerged it fine with it (upstream hasnt been updated in a while...) so to avoid LOTS of repoman errors, I added 3.6 to its python_compat. 

Hopefully everything is done correctly, python ebuilds are hard :)

### CODE REVIEW

File: dev-python/texttable/texttable-1.2.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+		dev-python/pytest[${PYTHON_USEDEP}]
+		cjk? ( dev-python/cjkwrap[${PYTHON_USEDEP}] )
+	)
+"
+RDEPEND="
+	cjk? ( dev-python/cjkwrap[${PYTHON_USEDEP}] )
+"
+
+python_test() {
+	py.test tests.py || die
```
Comment (mgorny) [inline dev-python/texttable/texttable-1.2.1.ebuild]: `-v`, please.

File: media-video/vcsi/vcsi-7.ebuild [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+inherit distutils-r1
+
+DESCRIPTION="Create thumbnail sheets from video files"
+HOMEPAGE="https://github.com/amietn/vcsi"
+SRC_URI="https://github.com/amietn/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+
```
Comment (mgorny) [inline media-video/vcsi/vcsi-7.ebuild]: Stray empty line.

File: media-video/vcsi/vcsi-7.ebuild [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+"
+RDEPEND="
+	dev-python/parsedatetime[${PYTHON_USEDEP}]
+	>=dev-python/texttable-1.0[${PYTHON_USEDEP}]
+	media-fonts/dejavu
+	media-video/ffmpeg
+"
+
+python_test() {
+	nosetests -v pytest tests || die "python tests failed with nose"
```
Comment (mgorny) [inline media-video/vcsi/vcsi-7.ebuild]: What does 'pytest' do here?

**Comment by mgorny** [comment]
(squash bugfixes over your new ebuilds)

**Comment by juippis** [comment]
Glad youre back. Thanks for the review! 

Everything should be fixed now, and squashed commits.

**Comment by mgorny** [comment]
All looks good, thanks!

**Pull request closed without merging**